Update install-previous-sync-app.md (#2767)
MicrosoftDocs/OfficeDocs-SharePoint: OneDrive/install-previous-sync-app.md

    c. If the **User Account Control** dialog box appears, confirm that the action displayed is what you want, and then select **Continue**.

> 5. At the command prompt, navigate to a folder you extracted Office Deployment tool into

Reviewer: ```suggestion
5. At the command prompt, run the following commands, and press **Enter** after each one:
```

    ```DOS

>     cd /d %userprofile%\desktop

Reviewer: ```suggestion
    cd /d %userprofile%\desktop
    setup.exe /configure AddODB.txt
```

>     ```
> 6. Run command below
> 
>     ```DOS
>     setup.exe /configure AddODB.txt
>     ```

Reviewer: ```suggestion
```

   
    > [!NOTE]
   > The installation occurs silently in the background and can take 20 minutes or more to complete, depending on the speed of your Internet connection. Don't close the Command Prompt window during the installation. When the installation is completed, a new command line appears at the command prompt.

> 7. Select **Start**. In the **Start Search** box, type **OneDrive**, and then start OneDrive.

Reviewer: ```suggestion
6. Select **Start**. In the **Start Search** box, type **OneDrive**, and then start OneDrive.
```

    > [!NOTE]
    > After you set up OneDrive, you might not see sync icons on files until you restart your computer.

> 8. Close the command prompt. You can now safely delete the following files from your desktop:

Reviewer: ```suggestion
7. Close the command prompt. You can now safely delete the following files from your desktop:
```
